ts/screens/authentication/cie/CiePinScreen.tsx

```diff
@@ -48,140 +56,163 @@ const styles = StyleSheet.create({
 });
 
 const CIE_PIN_LENGTH = 8;
+const FORGOT_PIN_PAGE_URL =
+  "https://www.cartaidentita.interno.gov.it/richiesta-di-ristampa/";
 
-/**
- * A screen that allow the user to insert the Cie PIN.
- */
-class CiePinScreen extends React.PureComponent<Props, State> {
-  private continueButtonRef = React.createRef<FooterWithButtons>();
-  private pinPadViewRef = React.createRef<View>();
-  constructor(props: Props) {
-    super(props);
-    this.state = { pin: "" };
-  }
+const CiePinScreen: React.FC<Props> = props => {
+  const { showAnimatedModal, hideModal } = useContext(LightModalContext);
+  const { navigate } = useContext(NavigationContext);
+
+  const onOpenForgotPINPage = () => openWebUrl(FORGOT_PIN_PAGE_URL);
 
-  private getContextualHelp = () => ({
+  const { present } = useIOBottomSheet(
+    <View>
+      <Body>{I18n.t("authentication.cie.pin.bottomSheetText")}</Body>
+      <Link onPress={onOpenForgotPINPage}>
```

Comment:
When a pressable item is inside the bottom sheet, in Android it is difficult to tap
see https://github.com/pagopa/io-app/blob/16416b3c865420ff1f8a98161e81173fe8ae836c/ts/components/ButtonDefaultOpacity.tsx#L40 
```suggestion
      <ButtonDefaultOpacity
        onPress={onOpenForgotPINPage}
        style={styles.bsLinkButton}
        onPressWithGestureHandler={true}
      >
        <Link>
          {I18n.t("authentication.cie.pin.bottomSheetCTA")}
        </Link>
      </ButtonDefaultOpacity>
      
      [...]
        bsLinkButton: {
    paddingRight: 0,
    paddingLeft: 0,
    marginVertical: 20,
    height: 60,
    backgroundColor: IOColors.white
  }
```

Reply:
Nice to know.